Launcher/Classes/ViewController.swift

```diff
@@ -684,5 +696,45 @@ class TasksViewController: NSViewController {
             strongSelf.createIRBSessionTask.waitUntilExit()
         }
     }
+    
+    func getCurrentDeviceUDID() -> String {
+        let simulatorUDIDs = "\(phoneComboBox.itemTitle(at: phoneComboBox.indexOfSelectedItem))"
+            .split(separators: "[]")
+            .map(String.init)
```

Comment:
Why not just ‘split’ then return ‘last’ as an optional String and handle the optional at the calling side?

Reply:
Not sure if this is intended - or what's actually happening here at all - but why do you return the second element in the array?

Reply:
Can we add a simple test for this function maybe?

Reply:
the extension of split allows to split the String by two elements "[" and "]" here. The goal was parsing simulator string (e.g. `iPhone 7 (11.0) [udid-xxx-xx] (Simulator)`). Can you suggest peter way to get the UDID? @BasThomas @q231950

Reply:
I think a regular expression will give you a good solution.

Reply:
Or something that gets all string between `[]` maybe?